code/game/objects/items/storage/toolbox.dm

```diff
@@ -41,6 +41,12 @@
 	user.visible_message("<span class='suicide'>[user] robusts [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
+/obj/item/storage/toolbox/attack(mob/M, mob/user)
+	..()
+	if(user.zone_selected == BODY_ZONE_HEAD)
+		if(prob(1))
+			playsound(loc, 'sound/items/bonk.ogg')
```

Comment:
won't this play alongside the normal toolbox strike sound?

Reply:
Yes